Services/CrontabInstaller.php

```diff
@@ -66,23 +68,24 @@ public function __construct(CrontabExecutor $executor, LoggerInterface $logger,
     /**
      * Install
      *
-     * @return void
+     * @return int
      */
-    public function install()
+    public function install(): int
     {
         $changed = $this->executor->dump();
 
         if ($changed) {
-            $this->executeInstall();
+            return $this->executeInstall();
         }
+        return Command::FAILURE;
```

Comment:
Are you sure that should return fail status? from my perspective, if config not changed it's not an error

Reply:
_dump_ before _install_ (w/o config changes in between) prevents crontabs from being installed (actually added to OS crontab). For me such behavior is definitely a failure of "**installation**".